ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -140,6 +140,14 @@ PMP_SS_RMM:
   - low_pressure_status
   - high_pressure_status
 
+PMP_PSM2X:
```

Comment:
The `PMP` entity type should refer to a single pump rather than a collection. Based on my comment above, I would suggest having two `PMP` entities that are both `PMP_SS_VSC_WDPM`. Please check if that would work for your application.

Reply:
fixed, yes i can make this work